ifrs17-template/Export/MapTemplate.ipynb

```diff
@@ -78,6 +78,37 @@
             "execution_count": 0,
             "outputs": []
         },
+        {
+            "cell_type": "markdown",
+            "source": [
+                "# Sharepoint file storage"
```

Comment:
Move all this new code to a dedicated NB called ExportSettings in Export directory and import it (ExportSettings) in all NB you export. 
In this way you avoid duplication in the LoadData NB (which should only load data)